tests/unit/backend/helpers/generate-form-data.ts

```diff
@@ -117,9 +123,9 @@ export const generateProcessedSingleAnswerResponse = (
       field.fieldType,
     )
   ) {
-    throw new Error(
+    return (new Error(
```

Comment:
is this really necessary? if this line of code is ever reached, it'll be much more confusing to the developer to debug a null pointer exception on an `Error` cast as a `ProcessedSingleAnswerResponse` than a simple `throw` which halts execution immediately on a specific line number